.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,6 +18,15 @@ Before we can approve your change; please submit the following in a comment:
 
 <!--  Doc PR (not required but highly recommended) -->
 
+**Backporting**
+<!--
+PRs targeting the default master branch will go into the next major release usually.
+If this PR should be backported to the current or earlier releases then please submit a PR for that particular branch.
+For example, to merge a change to the 1.8.X series then provide a PR for the 1.8 branch.
+-->
+- [ ] Changes should be applied to another release.
```

Comment:
any stable branch will require this if we keep the current branching model. how about making this a expanding field

[ ] Backport to latest stable release? 

Also, delete references to 1.8?

Reply:
Sure, I wanted a specific example hence the 1.8 but will remove.